src/azul/chalice.py

```diff
@@ -526,6 +533,33 @@ def metric_thresholds(self) -> list[MetricThreshold]:
                                                       value=threshold))
         return thresholds
 
+    def retry(self, retries: int):
+        """
+        Use this decorator to specify a custom number of retries that should be
+        different from the default (which is two) for any of the Azul async
+        lambdas. See:
+        https://docs.aws.amazon.com/lambda/latest/dg/invocation-retries.html
+        """
+
+        def wrapper(f):
+            assert isinstance(f, chalice.app.EventSourceHandler), f
+            f.retries = retries
+            return f
+
+        return wrapper
+
+    def retries(self) -> list[Retry]:
+        lambda_name, _ = config.unqualified_resource_name(self.app_name)
+        retries = []
+        for handler_name, handler in self.handler_map.items():
+            if isinstance(handler, chalice.app.EventSourceHandler):
```

Comment:
As discussed in PL, this conflates the case in which the inner decorator was a no-op for whatever reason and the case in which the developer ordered the decorators incorrectly.

Reply:
I also think that we should make our choice of 2 explicit for the forwarder functions.